Services/DataBaseService/Database/TPlatformDbContext.cs

```diff
@@ -10,6 +10,7 @@ public class TPlatformDbContext : DbContext
     {
         public DbSet<DbUser> Users { get; set; }
         public DbSet<DbUserCredential> UsersCredentials { get; set; }
+        public DbSet<DbUsersAvatars> UserAvatars { get; set; }

```

Comment:
It would be better if you wrote this material in the same style with other names